src/order-history/OrderHistoryPage.jsx

```diff
@@ -184,31 +184,44 @@ class OrderHistoryPage extends React.Component {
     const hasOrders = orders.length > 0;
 
     return (
-      <div className="page__order-history container-fluid py-5">
-        <h1>
-          {this.props.intl.formatMessage(messages['ecommerce.order.history.page.heading'])}
-        </h1>
-        {loadingError ? this.renderError() : null}
-        {loaded && hasOrders ? (
-          <>
-            <MediaQuery query="(max-width: 768px)">
-              {this.renderMobileOrdersTable()}
-            </MediaQuery>
-            <MediaQuery query="(min-width: 769px)">
-              {this.renderOrdersTable()}
-            </MediaQuery>
-            {this.renderPagination()}
-          </>
-        ) : null}
-        {loaded && !hasOrders ? this.renderEmptyMessage() : null}
-        {loading ? this.renderLoading() : null}
-      </div>
+      <section className="page__order-history">
+        {this.props.isB2CSubsEnabled ? (
+          <h2>
+            {this.props.intl.formatMessage(
+              messages['ecommerce.order.history.payments.heading'],
+            )}
+          </h2>
+        ) : (
+          <h1>
+            {this.props.intl.formatMessage(
+              messages['ecommerce.order.history.page.heading'],
+            )}
+          </h1>
+        )}
+        <div>
+          {loadingError ? this.renderError() : null}
+          {loaded && hasOrders ? (
+            <>
+              <MediaQuery query="(max-width: 768px)">
```

Comment:
Is there a reason to not use media queries in a `.scss` files?

Reply:
this code is existing I didn't change it, only showing in change-log because of the indentation change